app/jobs/audit_sign_in_event_job.rb

```diff
@@ -0,0 +1,23 @@
+require 'spreadsheet_writer'
+require 'message_encryptor'
+
+class AuditSignInEventJob < ApplicationJob
+  queue_as :audit_sign_in_event
+
+  def perform(audit_details)
+    return unless PUBLISHED_VACANCY_SPREADSHEET_ID
```

Comment:
You need to write these events to a different spreadsheet? Unless you're writing to a different tab on the same sheet, in which case can you rename the variable